dd-sdk-android-ndk/src/main/cpp/utils/signal-monitor.c

```diff
@@ -106,107 +189,57 @@ bool try_to_install_handlers(JNIEnv *env) {
         return false;
     }
 
-    return override_native_signal_handlers();
-}
+    sigemptyset(&signals_mask);
+    // Main thread does not block by default the SIGQUIT signals so we will have to
```

Comment:
Don't have any doc ...is just what I understood by studying the behaviour and seeing what others did. Fact is that if you just leave it as before and add the SIGQUIT signal in the list it will not be intercepted. That tells me that the Main Thread does not block (intercept) this signal so this is the reason I had to do this tweak and move everything on a new Thread with that signal masked (blocked) in order to intercept it and not affect the main thread.